images/kcp-registrar/register.sh

```diff
@@ -185,21 +185,38 @@ switch_ws() {
     KUBECONFIG="$kcp_kcfg" kubectl kcp workspace current
 }
 
+get_sync_target_name() {
+    local cluster_name="$1"
+
+    if [[ -z ${cluster_name} ]]; then
+        exit_error "Can't return SyncTarget Name because of missing cluster name!\n\n"
+    fi
+
+    local len=${#cluster_name}
+    if [ $len -le 43 ]; then
+        echo "$cluster_name"
+    else
+        echo "$(echo -n "$cluster_name" | md5sum | cut -d ' ' -f1)"
```

Comment:
The echo of the command output seems unnecessary. I would expect `echo -n "$cluster_name" | md5sum | cut -d ' ' -f1` to work just as well.

Reply:
Thanks. Fixed